static/js/operation.js

```diff
@@ -123,75 +123,76 @@ function parseAssembly(text, data, minContigLength) {
   var cols = ['id', 'length', 'gc', 'coverage']; // the information available in the contig titles
   var df = [];
 
-  var id = 0;
+  var id = NaN;
```

Comment:
`NaN` means "not a number". But `id` won't be a number any way. Although not impacting the result, I think [`null`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/null) is more reasonable because it means absence of value.